docs/lib/graphqlapi/fragments/flutter/getting-started/10_preReq.md

```diff
@@ -1,7 +1,5 @@
 * [Install and configure Amplify CLI](https://docs.amplify.aws/cli/start/install)
-* A Flutter application targeting Flutter SDK >= 1.20 (stable version) with Amplify libraries integrated
+* A Flutter application targeting Flutter SDK >= 2.0.0 (stable version) with Amplify libraries integrated
```

Comment:
I think the update here needs to be made in a few other places. I saw it still in the auth, analytics, and datastore getting started guide. I think you got the others.

Reply:
Oh good catch. I wonder what I was searching for that didn't pick these up..